src/app/store/state/services.state.ts

```diff
@@ -0,0 +1,33 @@
+import { State, Action, StateContext, Selector } from '@ngxs/store';
+import { Injectable } from '@angular/core';
+import { ServiceModel } from '../../models/service.model';
+import { AddService } from './../actions/service.actions';
+
+export class ServicesStateModel {
+  services: ServiceModel[];
+}
+
+@State<ServicesStateModel>({
+  name: 'services',
+  defaults: {
+    services: [],
+  },
+})
+@Injectable()
+export class ServicesState {
+  @Selector()
+  static getServices(state: ServicesStateModel): ServiceModel[] {
+    return state.services;
```

Comment:
Why not destructuring?

Reply:
do you mean:
```suggestion
    return ...state.services;
```
what is the point for that? in this way i get observable with array of services and that is probably the best object for listing them in some "All services" view

Reply:
do you mean this:
```suggestion
    const { services } = state;
    return services;
```
or what?

Reply:
I mean `static getServices({ services }: ServiceStateModel): ...`. :^)